test/parallel/test-stream-push-strings.js

```diff
@@ -55,7 +55,7 @@ const ms = new MyStream();
 const results = [];
 ms.on('readable', function() {
   let chunk;
-  while (null !== (chunk = ms.read()))
+  while (chunk = ms.read())
```

Comment:
The `!== null` test should be kept to make the code more explicit.

Reply:
A couple of our current tests rely on the implicit version. Do you want those also be converted? Because I would argue having a consistent style is best.

Reply:
Yes, but I'd rather have that in a separate PR.